package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "uk.co.compucorp.civicase",
+  "version": "2.0.0",
+  "description": "Civicase Extension based on Shoreditch theme",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs",
+    "test": "tests"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "precommit": "lint-staged"
+  },
+  "lint-staged": {
+    "*.js": [
+      "semistandard --fix",
+      "git add"
+    ],
+    "*.scss": [
```

Comment:
Please remove ` "*.scss"` block for now, because we dont have scss yet. When we are going to switch to scss, we will add this.
I added it as part of Technical Steps by mistake.
Also please update the PR title accordingly.